### PR TITLE
Update changelog for release

### PR DESCRIPTION
**Pull request by da-kami**
Thought about actually creating the ticket for release, but noticed that we never updated the `CHANGELOG`.

Should we add more @luckysori @tcharding ?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 -   **Breaking Change**: Rename `parity` to `geth` in the configuration file as we are only testing the software with Geth as part of the CI.
+-   **Breaking Change**: Depricate rfc003 routes by adding `rfc003` to GET and POST paths.
```
Comment (thomaseizinger) [inline CHANGELOG.md:19]:
```suggestion
-   **Breaking Change**: Deprecate rfc003 routes by adding `rfc003` to GET and POST paths.
```

Comment (thomaseizinger) [reply CHANGELOG.md:19]: Wait, what is happening here? The paths have been designed from the beginning to be purely additive? Why did we have to make these changes?

Comment (da-kami) [reply CHANGELOG.md:19]: Let me check if I actually added this correctly to the changelog 😬

Comment (da-kami) [reply CHANGELOG.md]: My bad, there is not breaking change here. I removed the line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+
+-   Support for `herc20-halight`, `halight-herc20`, `hbit-herc20` and `herc20-hbit` swaps.
```
[the remark this answers is not in the document]
Comment (luckysori) [reply CHANGELOG.md:22]: I was just trying to be consistent with you, and assuming that the rest of the team was doing the same. I can't say I prefer it over `alpha-beta`, but I think it's fine! I'll let someone else decide this one :stuck_out_tongue:

**Pull request merged**